File: PyCpp/data/test2.java

import <iostream>
import <iostream.h>
import "parselib/local.hpp"
import <std/fstream>
import "parselib/local.h"

class myclass {

    /*!
    doc goes here
    */
    constructor();

    constructor(int a, const float b);

    /*!
    destruct comment
    */
    destructor();

    /*! \brief attr doxy doc */
	public @py const long &a2 ;
	private char *& c3;
	public string &&blob;

    /*! method documentation
    \param param3 description
    \return processed string
    */
	public const string b (const int param3 , string param2) @{
	    code goes here
	    lorem ipsum
	@}

	public @py int e (const int t);

	protected vector<pair<int, int, vector<int>, int>> e;
} ;

File: PyCpp/data/test_srcs/itf2/test2.cxx

///@module mymodule.test2

#include <iostream>

#include "../itf1/test1.h"

class Myclass2 : public mymodule::Player<T1, T2<V1, v2, V3> >, private otherClasdb, public XAz<T> {

    /*!
    doc goes here
    */
    constructor();

    constructor(int a, const float b);

    /*! \brief attr doxy doc */
	ns::int a2;
	float c3;

} ;